doc/source/getting_started/docker.rst

```diff
@@ -104,6 +105,10 @@ Verify your connection with:
 
 Additional Considerations
 -------------------------
+
+Appending MAPDL options to the container process
```

Comment:
```suggestion
Appending MAPDL Options to the Container
```

Reply:
@germa89 I will eventually redo all headings to sentence case, but currently PyMAPDL uses initial capital letters on key words. I don't think "Process" needs to be present in the heading, but correct me if I'm wrong!